mnemosyne/core/beam.py

```
@@ -4172,6 +4172,11 @@ def forget_working(self, memory_id: str) -> bool:
                     "DELETE FROM annotations WHERE memory_id = ?", (memory_id,)
                 )
                 cursor.execute("DELETE FROM memory_embeddings WHERE memory_id = ?", (memory_id,))
+                # gists table is optional (episodic graph extension) — skip if missing
+                try:
+                    cursor.execute("DELETE FROM gists WHERE memory_id = ?", (memory_id,))
+                except Exception:
+                    pass
```

Ruff (0.15.21)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)